hoomd/md/pair.py

```diff
@@ -2377,3 +2377,75 @@ def __init__(self, nlist, r_cut=None, r_on=0., mode='none'):
             TypeParameterDict(a=list, b=list,
             _defaults=dict(a=[float]*3, b=[float]*3), len_keys=2))
         self._add_typeparam(params)
+
+
+class OPP(Pair):
+    """Oscillating pair potential.
+
+    Args:
+        nlist (:py:mod:`hoomd.md.nlist.NList`): Neighbor list
+        r_cut (float): Default cutoff radius (in distance units).
+        r_on (float): Default turn-on radius (in distance units).
+        mode (str): energy shifting/smoothing mode
+
+    `OPP` specifies that an oscillating pair potential should be applied between
+    every non-excluded particle pair in the simulation. The OPP potential can
+    be used to model metallic interactions.
```

Comment:
Add a citation to the original paper that introduced this potential.

Reply:
Will do.